content/2016-03-15-lets-encrypt-and-nginx-state-of-the-art-of-a-secure-web-deployment.md

```diff
@@ -354,6 +355,10 @@ The report mode can be enabled using:
 
     Content-Security-Policy-Report-Only instead of Content-Security-Policy
 
+    add_header Referrer-Policy origin-when-cross-origin;
```

Comment:
Can you put this policy in its own sub-section, including the description and a link to securityheaders.io for its usage, like the other headers ?